clj-refactor.el

```diff
@@ -3653,7 +3661,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-create-fn-from-e
         (buffer-substring (point-min) (point-max)))
     s))
 
-(defun cljr--is-keyword? (s)
+(defun cljr--sans-ns-keyword? (s)
```

Comment:
This function should be `cljr--keyword?` because that's what we care about everywhere else.

You're right that the function is flawed, because it doesn't deal with `::aliased/keywords`, but the intention everywhere else is to just check for keywords, even though the implementation is slightly wrong atm.

I suggest we rename this back and just deal with the ns-checking in `cljr--in-keyword-sans-alias?`.  `cljr--symbol-prefix` can be used to check for an alias.